### PR TITLE
[Dashboard] Remove <AspectRatio/>

### PR DESCRIPTION
**Pull request by kien-ngo**
## Problem solved

Short description of the bug fixed or feature added

---

## PR-Codex overview
This PR primarily focuses on removing the `AspectRatio` component from various files and replacing it with a `div` that uses Tailwind CSS classes for aspect ratios. Additionally, it updates the `package.json` to remove the `@radix-ui/react-aspect-ratio` dependency.

### Detailed summary
- Deleted `AspectRatio` component from multiple files.
- Replaced `AspectRatio` with `div` using Tailwind CSS aspect ratio classes.
- Updated `package.json` to remove `@radix-ui/react-aspect-ratio` dependency.
- Added `AspectRatioProps` to ESLint configuration.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by kien-ngo** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4750?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4750** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4750?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4749** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4749?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4748** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4748?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kien-ngo and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/(chainPage)/components/server/chain-header.tsx [hunk 1]
```diff
@@ -1,4 +1,3 @@
-import { AspectRatio } from "@/components/ui/aspect-ratio";
```
Comment (kien-ngo) [inline apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/(chainPage)/components/server/chain-header.tsx:1]: @jnsdls I didn't know there was another AspectRatio so I mistakenly removed this one. And now the lint is telling me due to this change, `@radix-ui/react-aspect-ratio` is no longer being used.

So, should we remove said dependency, or use in as the replacement for all the Chakra-ui's AspectRatio?

Comment (kien-ngo) [reply apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/(chainPage)/components/server/chain-header.tsx:1]: (I have a feeling you would want to remove the dep)

Comment (jnsdls) [reply apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/(chainPage)/components/server/chain-header.tsx:1]: @MananTank had thoughts on this last time it came up, will have him make the decision on what we do here, but either way it's either we use css or we use the component, we have to pick one of the options, not both

Comment (MananTank) [reply apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/(chainPage)/components/server/chain-header.tsx:1]: We can replace with it with CSS/tailwind class here but my only concern is that it's not a an exact replacement, (I've seen this when I refactored the FileInput component - I had to make some other changes to make the aspect-ratio work exactly as chakra version ). 

Lets verify that all changes are working as expected and we are not accidentally breaking the layout

Comment (kien-ngo) [reply apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/(chainPage)/components/server/chain-header.tsx:1]: @MananTank yeah it's not working exactly the same you're right. I had to make few minor changes to re-align the inner content (re-center, more like).

I guess that means we stick with `@radix-ui/react-aspect-ratio`

Comment (MananTank) [reply apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/(chainPage)/components/server/chain-header.tsx:1]: Let's replace the instances where it works with just a css class and make changes in other places as needed and remove AspectRatio component

Comment (kien-ngo) [reply apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/(chainPage)/components/server/chain-header.tsx:1]: Tested this PR. should be gud

File: apps/dashboard/src/components/ipfs-upload/dropzone.tsx [hunk 2]
```diff
@@ -74,12 +74,13 @@ export const IpfsUploadDropzone: React.FC = () => {
...
-          base: droppedFiles.length ? 1 : 8 / 4,
-          md: droppedFiles.length ? 16 / 9 : 36 / 9,
-        }}
-        w="100%"
+      <div
+        className={cn(
+          "w-full",
+          droppedFiles.length
+            ? "aspect-square md:aspect-[16/9]"
+            : "aspect-[4/1] md:aspect-[36/9]",
```
Comment (jnsdls) [inline apps/dashboard/src/components/ipfs-upload/dropzone.tsx:82]: should be `8/4` for `base` no? which ... isn't that just `2` then?

**Review by jnsdls — CHANGES_REQUESTED** [review]

File: apps/dashboard/src/components/ipfs-upload/dropzone.tsx (hunk 2 above)
Comment (kien-ngo) [reply apps/dashboard/src/components/ipfs-upload/dropzone.tsx:82]: no way I just failed math

Comment (kien-ngo) [reply apps/dashboard/src/components/ipfs-upload/dropzone.tsx:82]: omg

**Review by jnsdls — APPROVED** [review]

**Pull request merged**